app/stores/UserStore.js

```diff
@@ -128,8 +111,17 @@ let UserStore = assign({}, BaseStore, {
         });
     },
 
-    getUsersByTeamId(teamId) {
-        return _teams_users[teamId] ? _teams_users[teamId] : [];
+    getUsersByTeamId(teamId, includeSub) {
+        let team = _teams[teamId];
+        let result = new Set(team.users || []);
+        if(!includeSub){
+            return Array.from(result);
+        }
+        team.teams.forEach(team =>{
+            let subResult = this.getUsersByTeamId(team.id, true);
+            result = new Set([...result, ...subResult]);
+        })
+        return Array.from(result);
     },
```

Comment:
Refer to https://github.com/rhinobird-io/rhinobird/commit/51d9268d1a219e60e7a404e473234e408ea70c3d#diff-d546a4c2cc41719deaaef6a7cac572caR275